### ISSUE TITLE
False negative in AtclauseOrder check

### ISSUE THREAD

**Issue by Vladlis** [checkstyle/checkstyle#2095]
Config:
```xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name = "Checker">
    <module name="TreeWalker">
         <module name="AtclauseOrder">
            <property name="tagOrder" value="@param, @return, @throws, @deprecated"/>
            <property name="target" value="CLASS_DEF, INTERFACE_DEF, ENUM_DEF, METHOD_DEF, CTOR_DEF, VARIABLE_DEF"/>
        </module>
    </module>
</module>
```
Input:
```java
    class SomeClass {
        /**
         * Some text.
         * @throws Exception Some text.
         * @serialData Some javadoc.
         * @return Some text.                 //violation
         */
        String method() throws Exception
        {
            return "null";
        }
    }
```
Result:
```bash
vlad@vlad-N61PC-M2S ~/cli $ java -jar checkstyle-6.10.1-all.jar -c my_check.xml Test.java
Starting audit...
Audit done.
```

**Comment by romani** [comment]
merged
